spec/lib/biran/configurinator_spec.rb

```diff
@@ -1,10 +1,37 @@
-require 'spec_helper'
-require 'biran'
+require "spec_helper"
+require "biran"
 
 describe Biran::Configurinator do
   subject { described_class.new }
 
-  it 'test' do
+  it "test" do
     expect(subject.files_to_generate).to eql({my_config: {extension: ".yml"}})
   end
+
+  describe "psych version" do
+    let(:data_content) { ERB.new(File.read("config/app_config.yml")).result }
+    let(:hash_content) { { "development" => {}, "test" => {} } }
+
+    context "when psych version is < 4.0" do
+      before(:example) do
+        stub_const("Psych::VERSION", "3.2")
+      end
+
+      it "calls right method" do
+        expect(YAML).to receive(:safe_load).with(data_content, [], [], true).and_return(hash_content)
```

Comment:
@night91 : so the `and_return` part is just sending back a return and we don't care about the output of the method call?  Meaning, we are only testing that we are sending the list of arguments to `safe_load` that we want, but don't care about what happens in the method?
Just trying to understand because that actually does what we want if so. I just want to make sure we aren't missing something else by ignoring the return... but I guess if we are specifically testing the arguments here we would test anything else later.

Reply:
Yes, it is as you said, the things is when you have loaded a pysch version greater or lower than 4, in one case it will work with some arguments or it will work with other arguments.

But what your change really did was providing the correct arguments to safe_load, not to actually now accept ing aliases, both calls had true for aliases.

Reply:
Correct, which I think your change tests what we want in that scenario.
I added some other tests... I think, that should check that aliases and nested aliases are working.

Can you review, I think I am done with what I want to accomplish in this PR. The testing environment has to be greater than 2.6 due to the Psych versions being supplied in the versions before that don't include the keyword argument. Too difficult to try to come up with a test scenario for a situation we don't want to support going forward. Same with not testing in 3.1. I know this will work in 3.1, I just can't test it do to dependencies in the test environment. I will work on that in another PR and then properly setup some boundaries of required rubies and things like that. 

This code should still work in all the required environments we need at this point so we can do a point release with it as I don't think it will break anything.

Want to try to maybe include a few security updates as well before moving on in separate work. Please let me know your thoughts on my changes and additions to the tests.